packages/create-vite/src/index.ts

```diff
@@ -470,6 +514,42 @@ async function init() {
     template = variant
   }
 
+  // 5. Ask about immediate install and package manager
+  let immediate = argImmediate
+  // If agent is specified but immediate is not explicitly set, default immediate to true
+  if (argAgent && immediate === undefined) {
+    immediate = true
+  }
+  if (immediate === undefined) {
+    // In test mode, default to false to avoid prompts
+    if (process.env._VITE_TEST_CLI) {
+      immediate = false
+    } else {
+      const immediateResult = await prompts.confirm({
+        message: 'Install and start now?',
+      })
+      if (prompts.isCancel(immediateResult)) return cancel()
+      immediate = immediateResult
+    }
+  }
+
+  const pkgManager = pkgInfo ? pkgInfo.name : 'npm'
+  let agent = argAgent
+  if (immediate && !agent) {
+    const agentResult = await prompts.select({
+      message: 'Select a package manager:',
+      options: [
+        { label: 'npm', value: 'npm' },
+        { label: 'yarn', value: 'yarn' },
+        { label: 'pnpm', value: 'pnpm' },
+        { label: 'bun', value: 'bun' },
+      ],
+      initialValue: pkgManager,
+    })
```

Comment:
Perhaps here there's an opportunity to combine as a single prompt for most cases. Could we ask in the first prompt something like this?

```
> Install with <detected-pm> and start now?
- Yes
- Select a different package manager
- No
```

Alternatively I'm also thinking we could skip the package manager prompt altogether and rely on the one used to execute `create-vite`. We already provided the [package manager-specific `create-vite` commands](https://vite.dev/guide/#scaffolding-your-first-vite-project), and worst case if they forgot to change they'd just have to manually execute the commands.